h2o-r/tests/testdir_parser/runit_PUBDEV_5705_drop_columns_parser_arff_large.R

```diff
@@ -60,14 +60,6 @@ test.parseSkippedColumnsARFF<- function() {
   skipall <- onePermute
   skip99Per <- onePermute[1:floor(h2o.ncol(f1) * 0.99)]
 
-  # test skipall for h2o.importFile
-  e <-
-    tryCatch(
-      assertCorrectSkipColumns(fileName, fullFrameR, skipall, TRUE, h2o.getTypes(f1)),
-      error = function(x)
-        x
-    )
-  print(e)
   # test skipall for h2o.uploadFile
```

Comment:
this looked fine to me, could be kept or not, was mainly wondering why the first 16 lines of this file all look the same.
and why this function test.parseSkippedColumnsARFF looks copy/pasted multiple times later in this file

Reply:
I planned to do the following:

1. skipped all columns (this should generate an error)
2. skipped 99% of the columns.

Then, I want to test both h2o.uploadFile and h2o.importFile.

Based on this calculation, I will need to have 4 parseSkippedColumnsARFF since the actual tests are performed in this function.